autocomplete/definitions/global/tes3/getKeyName.lua

```diff
@@ -0,0 +1,11 @@
+return {
+	type = "function",
+	description = "Gets the name of a corresponding `tes3.scanCode`, using the appropriate GMSTs. The `keyName` returned by this function is the same `keyName` \z
+		that would be used in the in-game Controls menu. \z
```

Comment:
IMHO, it would look nicer on the page if you either removed the newline after `\z` or added another one.

Reply:
There isn't currently a line break being rendered on the page, but I agree that it does look a bit awkward the way it is now. I'll add a line break and that should make it look nicer.